### ISSUE TITLE
Displaying "Free" price when is no price set. 

### ISSUE THREAD

**Issue by ProjectKarol** [strangerstudios/pmpro-woocommerce#23]
Hi, I found the bug
When plugin is activated and price in product is "no set" the plugin change product page and show add to cart button and price "free" . I try use jetpack woocomerce  plugin to set "call for price" when price is no set in product page. But always when plugin is active, price shows "free" and add to cart button appear.
I working right now to resolve this issue, but I afraid, about my skills .
 Anyway when I found solution I will write about it.

( sorry for my poor English)

**Comment by ProjectKarol** [comment]
Ok , I am not developer  but I found solution how to fix issue what I mentioned above.
I just delete the code form the line  306 to  309 . 
This is the code which I deleted:
 // only change price if this is on the front end
if (!is_admin() || defined('DOING_AJAX')) {    
	add_filter("woocommerce_get_price", "pmprowoo_get_membership_price", 10, 2);
} 

I now this is very dirty way, but I am a nob  and this is only way to work form be. 
I hope this maybe helpful somehow.
